config-coral.ini

```diff
@@ -58,8 +58,8 @@ Device = EdgeTPU
 ; posenet use Coral's pose estimator to extract body's bounding boxes.
 Name = posenet
 ;ImageSize should be 3 numbers seperated by commas, no spaces: 300,300,3
-;For PoseNet model use: 1281,721,3
-ImageSize = 1281,721,3
+;For PoseNet model you can use: [1281,721,3], [641,481,3] or [481,353,3]
+ImageSize = 641,481,3
```

Comment:
Do you think this size is a better default than the original?

Reply:
When I tested the posnet model with the original size (641,481,3) I wasn't able to have a fluent live-feed.
With this size the live-feed works correctly and the accuracy is still very hight.

Reply:
So the model with the biggest input is the fastest? :thinking:

Reply:
No, the model with the biggest size is the slowest, but it has better accuracy.